contracts/HolographFactory.sol

```diff
@@ -133,25 +137,47 @@ contract HolographFactory is Admin, Initializable {
     require(!_isInitialized(), "HOLOGRAPH: already initialized");
     (address holograph, address registry) = abi.decode(data, (address, address));
     assembly {
-      sstore(0x5705f5753aa4f617eef2cae1dada3d3355e9387b04d19191f09b545e684ca50d, origin())
-
-      sstore(0x1eee493315beeac80829afd0aaa340f3821cabe68571a2743478e81638a3d94d, holograph)
-      sstore(0x460c4059d72b144253e5fc4e2aacbae2bcd6362c67862cd58ecbab0e7b10c349, registry)
+      sstore(_adminSlot, origin())
+      sstore(_holographSlot, holograph)
+      sstore(_registrySlot, registry)
     }
     _setInitialized();
     return IInitializable.init.selector;
   }
 
+  function bridgeIn(
+    uint32,
+    /* fromChain*/
+    bytes calldata payload
+  ) external returns (bytes4) {
+    (DeploymentConfig memory config, Verification memory signature, address signer) = abi.decode(
+      payload,
+      (DeploymentConfig, Verification, address)
+    );
+    deployHolographableContract(config, signature, signer);
+    return HolographableEnforcer.bridgeIn.selector;
+  }
+
+  function bridgeOut(
+    uint32,
+    /* toChain*/
+    address,
+    /* sender*/
+    bytes calldata payload
+  ) external pure returns (bytes4 selector, bytes memory data) {
+    return (HolographableEnforcer.bridgeOut.selector, payload);
+  }
+
   /**
    * @dev A sample function of the deployment of bridgeable smart contracts.
    * @dev The used variables and formatting is not the final or decisive version, but the general idea is directly portrayed.
    * @notice In this function we have incorporated a secure storage function/extension. Keep in mind that this is not required or needed for bridgeable deployments to work. It is just a personal development choice.
    */
   function deployHolographableContract(
-    DeploymentConfig calldata config,
-    Verification calldata signature,
+    DeploymentConfig memory config,
```

Comment:
We enabled bridgeIn for HolographFactory, this resulted in the `deployHolographableContract` to be called internally, which required to temporarily switch to memory instead of calldata in order to be able to easily pass data over.